File: Day 1/two_sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        for z in range(0, len(nums)):
            for i in range(z + 1, len(nums)):
                if (nums[z] + nums[i]) == target:
                    return [z, i]

File: Day 1/two_sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        int x = 0;
        int max = nums.size();
        for (x; x < max; x++) {
            for (int y = x + 1; y < max; y++) {
                if ((nums[x] + nums[y]) == target) {
                    return {x, y};
                }
            }
        }
        return {};
    }
};
